app/controllers/records_controller.rb

```diff
@@ -2,6 +2,7 @@ class RecordsController < ApplicationController
   include RecordsControllerBehavior
 
   def ingest_options
+    authorize! :new, GenericAsset
```

Comment:
I feel like this line checks with cancan if the current user can create a new asset, but why add it here?